flow/connectors/clickhouse/normalize.go

```diff
@@ -233,7 +233,11 @@ func getOrderedOrderByColumns(
 
 	orderbyColumns := make([]string, len(orderby))
 	for idx, col := range orderby {
-		orderbyColumns[idx] = getColName(colNameMap, col.SourceName)
+		sortingKeyName := col.SourceName
+		if col.SortingKeyExpression != nil && *col.SortingKeyExpression != "" {
```

Comment:
checking for both defeats the purpose of an `optional string`
either switch to just `string` or ensure you don't send empty string from upstream

Reply:
I prefer not using optional